ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -904,3 +835,330 @@ def test_ipahealthcheck_trust_agent_member_check(self):
         for check in data:
             assert check["result"] == "SUCCESS"
             assert check["kw"]["key"] == self.master.hostname
+
+
+@pytest.fixture
+def modify_permissions():
+    """Fixture to change owner, group and/or mode
+
+       This can run against multiple files at once but only one host.
+    """
+
+    state = dict()
+
+    def _modify_permission(host, path, owner=None, group=None, mode=None):
+        """Change the ownership or mode of a path
+        """
```

Comment:
ending `"""` should be on the same line for single line doc string as per pep8 standards.

Reply:
fixed.